ansible/roles/ci/doc/setup/semaphore.md

```diff
@@ -0,0 +1,29 @@
+# Semaphore Server
+
+## How to setup
+
+The following repository can be used as an example of [Semaphore](https://semaphoreci.com/) configuration [build servers check](https://github.com/shadow-robot/build-servers-check)
+
+The process of setting up [Semaphore](https://semaphoreci.com/) server
+
+  * In the root folder of your repository
+    * Create *repository.rosinstall* in case if you want to install some packages from source code. This file should be in in [rosinstall](http://wiki.ros.org/rosinstall) format.
+      You can use *repository.rosinstall* [build servers check](https://github.com/shadow-robot/build-servers-check) as an example.
```

Comment:
I think the identation of the Create is not needed.

"  * In the root folder of your repository, create *repository.rosinstall* in case you want to install some packages from source code. This file should be in [rosinstall](http://wiki.ros.org/rosinstall) format.
      You can use *repository.rosinstall* [build servers check](https://github.com/shadow-robot/build-servers-check) as an example."